src/pages/ProductList/Component/Dropdown/DropdownList.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import './DropdownList.scss';
+
+const DropdownList = props => {
```

Comment:
아래에서 사용되는 데이터를 컴포넌트의 매개변수 자리에서 구조분해할당하여 사용한다면, `props.`를 일일이 붙여주지 않아도 됩니다!
```suggestion
const DropdownList = ({ visibility, children }) => {
```

Reply:
수정 하였습니다.